app/api/V2/views/parcelviews.py

```diff
@@ -58,184 +58,61 @@ def get(self):
-            ]
-        }
-
-
-class DeclineParcel(Resource):
-    """Admin class for declining parcel order."""
-
-    def put(self, id):
-        """Put method to change parcel status to declined."""
-        order = Order().get_by_id(id)
-
-        if order:
-
-            if order.status != "pending":
-                return {"message": "order already {}".format(order.status)}
-
-            order.status = "declined"
-            return {"message": "Order declined"}
-
-        return {"message": "Order not found"}, 404
-
-
-class DeliveredParcels(Resource):
-    """Return a list of parcel orders completed by admin."""
-
-    def get(self):
-        """Get method to fetch all deliverd parcels."""
-        return {"completed orders": [order.serialize() for order in orders if
-                order.status == "completed"]}, 200
-
-
-class PendingParcels(Resource):
-    """Return all pending parcels."""
-
-    def get(self):
-        """Get method fethes all pending parcels."""
-        return {"Pending parcels": [order.serialize() for order in orders if
-                order.status == ("pending" or "Pending")]}, 200
-
-
-class MovingParcels(Resource):
-    """Class to handle orders in transit(moving)."""
-
-    def get(self):
-        """Return all moving orders."""
-        return {"moving parcels": [order.serialize() for order in orders if
-                order.status == "In Transit"]}
-
-
-class MarkParcelInTransit(Resource):
-    """Admin class for marking parcel status to moving."""
-
-    def put(self, id):
-        """Put method change status to moving."""
-        order = Order().get_by_id(id)
-
-        if order:
-            if order.status == ("completed" or "declined"):
-                return {"You already marked the order as {}"
-                        .format(order.status)}, 200
-
-            if order.status == "pending":
-                return {"message": "order should be approved first"}, 200
-
-            if order.status == "approved":
-                order.status = "moving"
-                return {"message": "Parcel {}"
-                        .format(id) + " now on road"}, 200
-
-        return {"message": "order not found"}, 404
-
-
-class ApproveParcel(Resource):
-    """Admin class for approving parcel."""
-
-    def put(self, id):
-        """Put method change parcel status to approved."""
-        order = Order().get_by_id(id)
+            current_orders.append(order)
+            return{"All orders":[order.serialize_order() for order in current_orders]}, 200
 
+class ChangeParcelDestination(Resource):
+    """class for changing parcel destination."""
+    @jwt_required
+    def put(self, order_id):
+        """Put method changes parcel destination."""
+        parser=reqparse.RequestParser()
+        parser.add_argument("destination",help ="Invalid name check again", required=True)
+        data=parser.parse_args()
+
+        query = "UPDATE orders SET destination = %s WHERE order_id = %s"
```

Comment:
this query should be in the model. Based on MVC the model should be the one to return the data to the Views